systemuicontroller/src/main/java/com/google/accompanist/systemuicontroller/SystemUiController.kt

```diff
@@ -173,8 +174,10 @@ fun rememberSystemUiController(): SystemUiController {
 internal class AndroidSystemUiController(
     private val view: View
 ) : SystemUiController {
-    private val window = view.context.findWindow()
-    private val windowInsetsController = ViewCompat.getWindowInsetsController(view)!!
+    private val window = requireNotNull(view.context.findWindow()) {
+        "The Compose View must be hosted in an Activity with a Window!"
+    }
```

Comment:
I made the `requireNotNull` explicit here on the window, since we were effectively requiring that anyway with the old code (at least on API 29 and below). This simplified a bit of the logic here as well.